src/Dropdown/index.jsx

```diff
@@ -48,9 +54,38 @@ const Dropdown = React.forwardRef(
       }
     };
 
-    return <BaseDropdown show={internalShow} onToggle={handleToggle} {...rest} ref={ref} data-testid="dropdown" />;
+    return (
+      <BaseDropdown
+        className={classNames(
+          `pgn__dropdown-${variant}`,
```

Comment:
nit: maybe also add `pgn__dropdown` as the primary paragon-namespaced class name in addition to the variant one.